xml/installation-installation-configure_3par.xml

```diff
@@ -125,29 +125,23 @@
  <section xml:id="sec.3par-multipath">
   <title>Multipath Support</title>
   <para>
-   If you want to enable multipath for Cinder volumes carved from 3PAR FC/iSCSI
-   storage system, please go through the
+   Setting up multipath support is highly recommended for 3PAR FC/iSCSI
+   backends, and should be considered a default best practice.
```

Comment:
"We recommend setting up multipath support for 3PAR FC/iSCSI backends as default best practice." just a rewording suggestion to cut it down a bit.

Reply:
Done